src/main/groovy/de/dkfz/roddy/bam2fastq/BamToFastqWorkflow.groovy

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (c) 2018 DKFZ - ODCF
+ *
+ * Distributed under the MIT License (license terms are at https://github.com/TheRoddyWMS/BamToFastqPlugin/blob/master/LICENSE.txt).
+ */
+package de.dkfz.roddy.bam2fastq
+
+import de.dkfz.roddy.Roddy
+import de.dkfz.roddy.config.ConfigurationError
+import de.dkfz.roddy.core.DataSet
+import de.dkfz.roddy.core.ExecutionContext
+import de.dkfz.roddy.core.ExecutionContextError
+import de.dkfz.roddy.core.Workflow
+import de.dkfz.roddy.execution.io.ExecutionService
+import de.dkfz.roddy.knowledge.files.BaseFile
+import de.dkfz.roddy.knowledge.files.FileGroup
+import de.dkfz.roddy.tools.LoggerWrapper
+
+/*
+ * Copyright (c) 2018 DKFZ - ODCF
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+import groovy.transform.CompileStatic
+
+import java.nio.file.Files
+
+@CompileStatic
+class BamToFastqWorkflow extends Workflow {
+
+    public static final LoggerWrapper logger = LoggerWrapper.getLogger("BamToFastqWorkflow")
```

Comment:
I wonder, if we could create an annotation class which automatically provides a logger? Any ideas?
Something like
``` Java
@WithLogger
class...
```